src/main/resources/db/migration/0200/V249__ProjectVoting.sql

```diff
@@ -0,0 +1,27 @@
+-- Add vote options enum table (YES, CONDITIONAL, NO)
+CREATE TABLE accelerator.vote_options (
+    id INTEGER PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    value TEXT NOT NULL,
```

Comment:
Call this column `name` -- our code generation configuration expects enum tables to have `id` and `name` columns.